xonsh/environ.py

```diff
@@ -710,7 +711,7 @@ def locate_binary(name):
 
     try:
         return next(chain.from_iterable(yield_executables(directory, name) for
-                    directory in builtins.__xonsh_env__.get('PATH') if os.path.isdir(directory)))
+                    directory in ([_get_cwd()] + builtins.__xonsh_env__.get('PATH')) if os.path.isdir(directory)))
```

Comment:
Points well taken.  I happy if the CWD change is only applied to Win if that is what people expect.